<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWidget</class>
 <widget class="QWidget" name="UsersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Użytkownicy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="guiIconsResources.qrc">
    <normaloff>:/icons/view-pim-contacts.png</normaloff>:/icons/view-pim-contacts.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Użytkownicy</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="3">
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="countLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Usuń</string>
          </property>
          <property name="icon">
           <iconset resource="guiIconsResources.qrc">
            <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="clientRadio">
        <property name="text">
         <string>Klient</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="workerRadio">
        <property name="text">
         <string>Pracownik</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="guiIconsResources.qrc"/>
 </resources>
 <connections/>
</ui>
